ui/components/component-library/modal/modal.scss

```diff
@@ -0,0 +1,6 @@
+.mm-modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  // z-index: 1040; // TODO: Check if this is required for Modal to work properly. It aligns with modal z-index in extension
```

Comment:
Nit: This will need removal.

Reply:
Good catch! Will need to address this during the migration to the new component once this has been merged. Removed in https://github.com/MetaMask/metamask-extension/pull/18156/commits/627858451d3098088fe9193d1d21824f56ef614a